var/spack/repos/builtin/packages/py-pypinyin/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPypinyin(PythonPackage):
+    """FIXME: Put a proper description of your package here."""
```

Comment:
Needs a description. Also can't contain unicode chars so you can't just copy the Chinese description lol.

Reply:
Should I just use the google translate version of that short description?

Reply:
That's fine with me